source/reference/parameters.txt

```diff
@@ -22,7 +22,7 @@ using:
 
      db.adminCommand( { setParameter: 1, <parameter>: <value>  } )
 
-- the :setting:`setParameter` configuration setting:
+- the :setting:`setParameter` configuration setting in the boot time configuration file:
```

Comment:
I think there is only ever one configuration time, hence adding "boot time" just seems confusing to me.

Reply:
This was my ignorance.  I had the idea in my head that the parameters were somehow stored somewhere independently of the startup command line (like other databases I use).  I'd like to withdraw this suggested edit.